### ISSUE TITLE
Error running ANY of the templates. 

### ISSUE THREAD

**Issue by AsherNoor** [d4emonsAVI/conpot#8]
**Describe the bug**
Conpot provides you with templates to start, including a 'default' template. When trying to run said template, the programs gives an error.

**To Reproduce**
1. Ran the virtual env (Python 3.6)
2. Run the command:  conpot -f
This provides you with the different templates.
3.  Run the command: conpot --t default
4. You receive the error.

**Expected behavior**
For the Siemens S7-200 ICS template to run.

**Desktop (please complete the following information):**
 - VM-OS: Windows 11
 - WSL : Ubuntu 18
 - Python virtual evn: 3.6

**Additional context**
**The Error:**

```

(conpot) adhd@DESKTOP-I1T2G01:~/ryno/bin$ conpot -t default
/home/adhd/conpot/lib/python3.6/site-packages/pyghmi/ipmi/private/session.py:31: CryptographyDeprecationWarning: Python 3.6 is no longer supported by the Python core team. Therefore, support for it is deprecated in cryptography. The next release of cryptography will remove support for Python 3.6.
  from cryptography.hazmat.backends import default_backend
WARNING:scapy.loading:Cannot read wireshark manuf database
WARNING:scapy.runtime:No route found for IPv6 destination :: (no default route?)
/home/adhd/conpot/lib/python3.6/site-packages/scapy/layers/ipsec.py:459: CryptographyDeprecationWarning: Blowfish has been deprecated
  cipher=algorithms.Blowfish,
/home/adhd/conpot/lib/python3.6/site-packages/scapy/layers/ipsec.py:473: CryptographyDeprecationWarning: CAST5 has been deprecated
  cipher=algorithms.CAST5,

                       
   ___ ___ ___ ___ ___| |_
  |  _| . |   | . | . |  _|
  |___|___|_|_|  _|___|_|
              |_|

  Version 0.6.0
  MushMush Foundation

Invalid arguments supplied. Please check that you pass both template and config arguments before running Conpot
```

**Comment by AsherNoor** [comment]
I also tried this command:
`conpot --template default`
Got the same error.

**Comment by AsherNoor** [comment]
It looks like the error happens with all the templates, giving the same error.

**Comment by AsherNoor** [comment]
Researching the error, and reading it again it seems that there are multiple things at play here.
The first stand-out issue, the environment is running Python 3.6.

The cryptography error flat out says that their newer versions will not support Python 3.6., and that 'CAST5' has been deprecated.
The version installed is 40.0.2.

The Scapy error is having issues reading databases, gives runtime errors, and mentions that "blowfish" has been deprecated. One solution proposed was to update to Scapy 2.5.0. 
That didn't work, it actually broke the conpot, and the only way to bring it back was to roll back scapy to 2.4.3rc1.

Still researching the issue.

**Comment by GryphonInfoSec** [comment]
You must pass a config argument. If you don't want to create one you must use the default (-f).
ie. 
$ conpot -f --template default

**Comment by AsherNoor** [comment]
@GryphonicInfoSec  was correct.
Once I was in the 'bin' directory I ran this command.

`conpot -f -t default`

And that started the template, and I was able to navigate to one of the servers by opening a browser and  typing:

`http://localhost:port/`

It is up and running.

![Conpot_Default-Template-Running_June14-2023](https://github.com/d4emonsAVI/conpot/assets/60714000/2899de81-afe7-491a-9914-cbc94f58c77d)
